lib/Interpreter/Interpreter.cpp

```diff
@@ -240,6 +241,20 @@ namespace cling {
     if (m_OptLevel == -1)
       setDefaultOptLevel(getCI()->getCodeGenOpts().OptimizationLevel);
 
+    if (!isInSyntaxOnlyMode() && m_Opts.CompilerOpts.CUDAHost) {
+      // Using a buffer instead of file I/O in the CUDA host backend is enabled
+      // by a valid pointer.
```

Comment:
"valid pointer" where? To what? which pointer? Suggest:
```suggestion
      // by getCI()->getCodeGenOpts().CudaGpuBinaryBuffer being set.
```
if that is what you mean.